### ISSUE TITLE
Flexbox works only in FF?

### ISSUE THREAD

**Issue by ivanjaros** [zurb/foundation-sites#7149]
Hi,
I have implemented flexbox and it is working in Firefox but not in Chrome or IE. But demos in docs do work.
Do I need to implement some specific css or JS that I am not aware of?

**Comment by rafibomb** [comment]
The <a href=“http://foundation.zurb.com/forum/

”>Foundation forum</a> would be a great place to seek some help with troubleshooting what might be going on here. If it turns out to be a defect, you'll have some more info to provide here which will help prioritize and get it fixed.

We're trying to keep the issue list focused on confirmed defects in Foundation, so any help everyone can provide to differentiate between usage / troubleshooting and actual defects is appreciated!

**Comment by ivanjaros** [comment]
I found the problem - the demo has vendor prefixes for flex whereas the scss files do not have them. I guess it is a matter of building the scss files "properly" but maybe a note or warning is in order?

**Comment by ivanjaros** [comment]
Another thing I have noticed is that I did not specify that I was talking about gridbox and not grid as a whole(which is working) and it is still not working in Chrome or IE even with prefixes and it looks like this is caused by the fact that there are no size settings. According the documentation only "column" class is required for items in gridbox but on demo page they do have sizing classes. Without them the gridbox is broken. I would suggest it is a documentation issue but on the other hand the gridbox never required sizing classes on items, only on the parent so it's a bug.
